.mailmap

```diff
@@ -189,6 +189,7 @@
 *Dan <coolg49964@gmail.com>
 *Marc-Etienne M.Leveille <protonyc@gmail.com>
 *Ulrich Hecht <ulrich.hecht@gmail.com>
+1e9abhi1e10 <1e9abhi1e10@gmail.com>
```

Comment:
How do you want your name to be recorded in the AUTHORS file?

Reply:
My GitHub username is `1e9abhi1e10 `and my name is `Abhishek Patidar`
If there is a choice then I prefer my name to be recorded in the AUTHOR'S file.

Reply:
Then change this line to:
```
Abhishek Patidar <1e9abhi1e10@gmail.com> 1e9abhi1e10 <1e9abhi1e10@gmail.com>
```